Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "sleipnir"
-version = "0.1.9"
```

Comment:
I think you can just do 0.1.10, you didn't change anything that requires a minor bump?